charts/tempo-distributed/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: tempo-distributed
 description: Grafana Tempo in MicroService mode
 type: application
-version: 1.8.0
+version: 1.8.1
 appVersion: 2.3.1
```

Comment:
Do you want to bump the `appVersion` here also?

Reply:
@zalegrala We don't appear to have a Tempo 2.3.2 release yet, which is why I hadn't.